test/main.py

```diff
@@ -0,0 +1,19 @@
+import os
```

Comment:
Why is python code here?

Reply:
To make end-to-end test script more convenient, python scripts are used instead of shell scripts

Reply:
@ComradeProgrammer  I mean this project is for k8s and mostly developed by Go. Why use Python for tests instead of go tests? Is it some convention used by all k8s ecosystem projects?